src/plugins/minimal_scene/MinimalSceneRhiOpenGL.hh

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright (C) 2021 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+#ifndef IGNITION_GUI_PLUGINS_MINIMALSCENE_TEXTURENODERHIOPENGL_HH_
+#define IGNITION_GUI_PLUGINS_MINIMALSCENE_TEXTURENODERHIOPENGL_HH_
```

Comment:
`_TEXTURENODERHIOPENGL_HH_` -> `_MINIMALSCENERHIOPENGL_HH_`

Reply:
Updated in ca52977561c63bb615e01b25fa62b350645dfda6